src/vivarium_census_prl_synth_pop/results_processing/addresses.py

```diff
@@ -166,19 +189,26 @@ def get_city_map(
     # Load addresses data from artifact
     addresses = artifact.load(data_keys.SYNTHETIC_DATA.ADDRESSES).reset_index()
     # Get observer data to map
-    output_cols = [column_name, "state"]
-    city_data = obs_data.reset_index()[output_cols].drop_duplicates().set_index("address_id")
+    if column_name == "address_id":
+        state_col = "state"
+    else:
+        state_col = "employer_state"
+    output_cols = [column_name, state_col]
+    city_data = obs_data.reset_index()[output_cols].drop_duplicates().set_index(column_name)
 
-    for state in city_data["state"].str.lower().unique():
+    for state in city_data[state_col].str.lower().unique():
```

Comment:
If you do the `unique` before the `.str.lower()` you'll be doing the latter operation on far fewer rows.

Reply:
I had to do it this way because doing `unique` first makes it an array so I will not have string functionality.

Reply:
for state in map(str.lower, city_data[state_col].unique()):

Reply:
You could convert the np array to a Series and then have the `.str` functionality.